src/hooks/volunteer/data.ts

```diff
@@ -20,7 +33,8 @@ export const useVolunteerData = ({
   isSectioned,
   onlyUpcoming = true,
   titleDetail,
-  bookmarkable
+  bookmarkable,
+  filterVariables
```

Comment:
If you are changing something on this file then how about sorting it alphabetically?

Reply:
But I still think it should be alphabetised with an extra commit.